### PR TITLE
git cta added

### PR DESCRIPTION
**Pull request by ameer-greenbull**

### CODE REVIEW

**Comment by AbdallaEmadEldinAbdalla** [comment]
@ameer-greenbull Premium CTA is not exists on portfolio/smart allocation pages.

File: src/components/layout/Nav/Nav.tsx [hunk 1]
```diff
@@ -423,64 +501,65 @@ const Nav = () => {
...
+                  >
+                    {t("common:signup")}
+                  </Link>
+                  <Link
+                    href="/login"
+                    className="bg-grey-3 py-3 text-white text-sm rounded-md font-medium w-full text-center"
+                  >
+                    {t("common:login")}
+                  </Link>
+                  {locale&&<Col className="bg-grey-3 text-white text-sm rounded-md font-medium w-full items-center">
```
Comment (almujtabamohammed) [inline src/components/layout/Nav/Nav.tsx:555]: Remove comment if not needed

File: src/components/shared/PremiumButtons/PremiumButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+
+const PremiumButton = ({className}:{className?:string}) => {
+  return (
+    <PaymentModal
+        triggerButton={
+            <Button className={twMerge("flex gap-2 items-center justify-start lg:justify-center group py-3 px-6 lg:p-0 lg:border-b-0",className)}>
+                <PremiumIcon/>
+                <span className='text-sm font-semibold text-slate-300 group-hover:text-slate-100'>Premium</span>
```
Comment (almujtabamohammed) [inline src/components/shared/PremiumButtons/PremiumButtons.tsx:15]: Add translation

File: src/components/shared/PremiumButtons/PremiumButtons.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  )
+}
+
+export const GetPremiumButton = () =>{
+    return (
+        <PaymentModal
+            triggerButton={
+                <Button className="flex gap-2 items-center justify-center group px-4 py-3 hover:bg-orange-1 border-orange-1 border rounded-md w-fit h-fit">
+                    <PremiumIcon className='group-hover:text-black-1'/>
+                    <span className='text-sm font-semibold text-orange-1 group-hover:text-black-1 hidden md:block'>Get Premium</span>
```
Comment (almujtabamohammed) [inline src/components/shared/PremiumButtons/PremiumButtons.tsx:28]: Add translation

**Review by almujtabamohammed — CHANGES_REQUESTED** [review]

**Pull request merged**